coderdojochi/emailtemplates/_email-base.html

```diff
@@ -35,7 +35,7 @@
 
     {% if preheader %}
```

Comment:
Will this line work since it's not using django templating anymore?

Reply:
Nope. Updated.